jenkins/scripts/coverage/generate-index-html.py

```diff
@@ -6,9 +6,9 @@
 with open('out/index.csv') as index:
   index_csv = filter(lambda line: line, index.read().split('\n'))
 
+# noqa
```

Comment:
what does this mean?

Reply:
Reverted.  https://github.com/nodejs/build/pull/1399#discussion_r266726111 but we no longer need it.